x/params/client/cli/tx.go

```diff
@@ -36,6 +36,9 @@ Proper vetting of a parameter change proposal should prevent this from happening
 (no deposits should occur during the governance process), but it should be noted
 regardless. 
 
+If the type of "value" is a struct and has "omitempty" in the tag in its definition, 
```

Comment:
this paragraph can be summarized to something like "only non-empty fields will be updated"